mantis/simulation/routers/data.py

```diff
@@ -795,6 +797,8 @@ def new_pair(
         pool_value_in_usd=pool_value_in_usd,
         in_token_amount=in_token_amount,
```

Comment:
remove old one

in_token_amount = in_asset_amount.

just renamed for conistency.

token = asset, so using asset in all places